### ISSUE TITLE
Chat Section 

### ISSUE THREAD

**Issue by RushiDonga** [clubgamma/snapchat_app_clone#5]
Design the Chats Section having
1). around 10 items in the list of friends
2). In place of User-pic, place your favourite colour or an image
3). username
4). "NewSnap" text below the username.

Write the code in lib->Chats->chats.dart
If need to create more .dart files create files in lib->Chats

![chats](https://user-images.githubusercontent.com/65907580/135726647-a8efa060-d89a-4818-9ce3-728ff592ac54.jpeg)

**Comment by pri1311** [comment]
Hey! I believe as of now only UI is to be created, and everything needs to be static. I would like to give this a try.

**Comment by RushiDonga** [comment]
Yes, everything needs to be static and yes you can give it a try :)
